src/wallet/rpcwallet.cpp

```diff
@@ -2716,7 +2716,7 @@ UniValue mint(const UniValue& params, bool fHelp)
 
     if (fHelp || params.size() != 1)
         throw std::runtime_error(
-            "mint amount\n"
+            "mint <amount>(could not smaller than 0.1)\n"
```

Comment:
`mint <amount>(0.1, 0.5, 1, 10, 100)`

Reply:
We can put any value and it will automatically divide into supported mints. So that why it can be any value as long as it can multiply with the lowest denomination.

Reply:
Ah ok, sure. In that case, I would just like to change it to be grammatically correct:
`"mint <amount>(No less than 0.1)\n"`

Reply:
GREAT! thanks for correction.

Reply:
After discussion it's better to add to arguments description instead.